spec/unit/mupdf_spec.lua

```diff
@@ -137,6 +139,29 @@ describe("mupdf module", function()
                 md5.sumFile(simple_pdf_compare)
             )
         end)
+        it("should open a page, add contents to an existing annotation and write a new document", function()
+            local ffi = require("ffi")
+            local doc = M.openDocument(simple_pdf_out)
+            assert.is_not_nil(doc)
+            local page = doc:openPage(1)
+            assert.is_not_nil(page)
+            local annot = page:getMarkupAnnotation(ffi.new("float[8]", {
+                 70,  930,
+                510,  930,
+                510,  970,
+                 70,  970 }),
+                1)
+            page:updateMarkupAnnotation(annot, "annotation contents")
+            page:close()
+            doc:writeDocument(simple_pdf_out_annotated)
```

Comment:
This is probably missing a `doc:close()` (As does the existing test above, apparently :/).

Reply:
Thanks, I added the `doc:close()` to both tests.